GoogleForCreators/web-stories-wp
# Dashboard: Animated first two pages of Entertainment template

## Summary

This PR adds animations to the first two pages of the Entertainment templates.

## To-do

Add animations to other pages.

## User-facing changes

None.  Animations are disabled with a feature flag by default.

## Testing Instructions

1.) Go to `Dashboard.php` and set `enableAnimation` -> `true`.
2.) Go to "Explore Templates" and hover over the Entertainment template and see how the first two pages animate in the preview 🎉 

## Screenshot

Animation in preview:
![entertainment_anim](https://user-images.githubusercontent.com/40646372/84551052-3e2ca400-acc1-11ea-8fd5-1e7045871601.gif)

Page 1:
![ent_page_1](https://user-images.githubusercontent.com/40646372/84551053-408efe00-acc1-11ea-82df-10073708f6eb.gif)

Page 2:
![ent_page_2](https://user-images.githubusercontent.com/40646372/84551054-4258c180-acc1-11ea-8eb9-c5fafc182f97.gif)
